swagger/messaging/activity_message.json

```diff
@@ -0,0 +1,249 @@
+{
+  "openapi": "3.0.0",
+  "info": {
+    "title": "Activity Message",
+    "description": "A set of API endpoints used by the inTandem platform to manage\n the send a client an activity message",
+    "version": "3.0"
+  },
+  "servers": [
+    {
+      "url": "https://api.vcita.biz/",
+      "description": "API Gateway server"
+    }
+  ],
+  "components": {
+    "securitySchemes": {
+      "Bearer": {
+        "type": "http",
+        "scheme": "bearer"
+      }
+    },
+    "schemas": {
+      "Response": {
+        "type": "object",
+        "properties": {
+          "success": {
+            "type": "boolean"
+          },
+          "data": {
+            "type": "object",
+            "properties": {
+              "activityMessage": {
+                "$ref": "https://vcita.github.io/developers-hub/entities/messaging/activityMessage.json"
+              }
+            }
+          }
+        },
+        "required": [
+          "success",
+          "data"
+        ]
+      },
+      "ErrorResponse": {
+        "type": "object",
+        "properties": {
+          "success": {
+            "type": "boolean"
+          },
+          "errors": {
+            "type": "array",
+            "items": {
+              "type": "object",
+              "properties": {
+                "code": {
+                  "type": "string"
+                },
+                "message": {
+                  "type": "string"
+                }
+              }
+            }
+          }
+        },
+        "required": [
+          "success",
+          "error"
+        ]
+      },
+      "RequestDto": {
+        "type": "object",
+        "properties": {
+          "business_uid": {
+            "type": "string",
+            "description": "The business that is sending the message."
+          },
+          "staff_uid": {
+            "type": "string",
+            "description": "The staff that is sending the message."
+          },
+          "client_uid": {
+            "type": "string",
+            "description": "Client to send the message to."
+          },
+          "message_uid": {
+            "type": "string",
+            "description": "The entity's unique identifier."
+          },
+          "activity_type": {
+            "type": "string",
+            "enum": [
+              "invite"
+            ],
+            "description": "Activity category type from a closed list."
+          },
+          "activity_action": {
+            "type": "string",
+            "enum": [
+              "schedule"
+            ],
+            "description": "Action related to the activity type property."
```

Comment:
I don't understand what is the purpose of this. Please sit with Aviya and improve the description of this with more elaborated description on how its used. Are we only supporting schedule now?